src/reddit/reddit_post.ts

```diff
@@ -40,7 +39,17 @@ export default class RedditPost {
     return new RedditPost(title, url, content, subreddit, user, timestamp);
   }
 
-  /** COnvert the reddit post to a game notification.
+  public static mdFromReddit(text: string): string {
+    let fulltext = '';
+    // User links
+    fulltext = text.replace(/\/u\/([a-zA-Z0-9]+)/, '[/u/$1](https://reddit.com/user/$1)');
+    // Subreddit links
+    fulltext = text.replace(/\/r\/([a-zA-Z0-9]+)/, '[/r/$1](https://reddit.com/r/$1)');
+
+    return fulltext;
+  }
```

Comment:
Maybe this needs to be part of a `reddit-utils` file but I didn't want to introduce too many new things in this PR